File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/AppRoleFeatureModel.ts
import { AppRoleModel } from "./appRoleModel";
import { ApplicationFeatureModel } from "./applicationFeatureModel";
import { AppRoleFeaturePermissionModel } from "./appRoleFeaturePermissionModel";

export interface AppRoleFeatureModel {
    appRoleFeatueId: number;
    appRoleId: number | null;
    featueId: number | null;
    canRead: boolean | null;
    canWrite: boolean | null;
    canSearch: boolean | null;
    canPrint: boolean | null;
    fullAccess: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;

}

File: SofiForce.Models/Models/EntityModels/AppRoleFeatureModel.cs
using System;

namespace Models
{
    public class AppRoleFeatureModel
    {
        public int AppRoleFeatueId { get; set; }
        public int? AppRoleId { get; set; }
        public int? FeatueId { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}
